deployment/debian-package-x64/pkg-src/rules

```diff
@@ -39,12 +41,25 @@ override_dh_auto_test:
 override_dh_clistrip:
 
 override_dh_auto_build:
+	echo $(WEB_VERSION)
+	# Clone down and build Web frontend
+	mkdir -p $(WEB_TARGET)
+	wget -O web-src.tgz https://github.com/jellyfin/jellyfin-web/archive/v$(WEB_VERSION).tar.gz || wget -O web-src.tgz https://github.com/jellyfin/jellyfin-web/archive/master.tar.gz
```

Comment:
can we print a warning somewhere that `master` was used as a fallback? I even would like to see this fallback enabled explicitly and disabled by default...

Reply:
Well, this only matters if one is building a non-master, non-release branch. Which is indeed a downside of this method, but there is no way around it that I know of without some weird hacks. I mean I could add an echo or something about it, but a flag is probably not possible (unless you know of an easy way).

Reply:
For the record, I'm doing the same thing with CentOS to, so it becomes "the standard", that if the release doesn't exist it falls back to master.

Reply:
For posterity, I should mention too that I asked on Riot, and none of the main web developers tend to use these packages for testing, so the point is pretty moot outside of some edge cases.

Reply:
I just don't like the silent fallback mode... so an environment variable which can turn it on (set by a build script) would suite me just fine, so it won't fallback "by accident".

Reply:
I suppose an envvar is possible, but I don't see it being very useful most of the time, especially with phase 2 which is splitting all this out. I'm only falling back to make nightlies easier.